coordinator-common/src/main/java/org/apache/kafka/coordinator/common/metrics/CoordinatorRuntimeMetrics.java

```diff
@@ -14,9 +14,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.kafka.coordinator.group.metrics;
+package org.apache.kafka.coordinator.common.metrics;
 
-import org.apache.kafka.coordinator.group.runtime.CoordinatorRuntime.CoordinatorState;
+import org.apache.kafka.coordinator.common.runtime.CoordinatorRuntime.CoordinatorState;
 
 import java.util.function.Supplier;
 
```

Comment:
The comment on line 24 would be better if it just said "the coordinator runtimes".